packages/material-ui/src/Tooltip/Tooltip.js

```diff
@@ -455,6 +455,22 @@ const Tooltip = React.forwardRef(function Tooltip(props, ref) {
     ref: handleRef,
   };
 
+  if (process.env.NODE_ENV !== 'production') {
+    childrenProps['data-mui'] = true;
```

Comment:
We should add some verbosity and explain its purpose so that people don't get confused and use this attribute for CSS selectors. It should be obvious that this is internal and should not be used by library consumers.